draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -172,14 +172,15 @@
    HTTP has been the primary information transfer protocol for the World Wide
    Web since its introduction in 1990. It began as a trivial mechanism for
    low-latency requests, with a single method (GET) to request transfer of a
-   presumed hypertext document identified by a given pathname (HTTP/0.9).
+   presumed hypertext document identified by a given pathname. This original
+   protocol is now referred to as HTTP/0.9.
```

Comment:
Maybe reference <https://www.w3.org/Protocols/HTTP/AsImplemented.html>?

Reply:
I am pretty sure that we tried that years ago and the RFC editor refused it.

Reply:
I'd try again.  They took a less stable URL to a WHATWG spec recently.